package.json

```diff
@@ -42,9 +43,10 @@
     "@volar-plugins/vetur": "latest",
     "@vue/eslint-config-standard": "^4.0.0",
     "@vue/eslint-config-typescript": "^9.1.0",
-    "@vue/test-utils": "^1.3.5",
+    "@vue/test-utils": "^2.4.0-alpha.2",
```

Comment:
needed to install `@vue/test-utils@next` to get compatible version of `vue-test-utils` with vue 3